### PR TITLE
Create GuildEducation.com.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

File: src/chrome/content/rules/GuildEducation.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	Invalid Certificate:
+		pages.guildeducation.com
+-->
+<ruleset name="GuildEducation.com">
+	<target host="guildeducation.com" />
+	<target host="www.guildeducation.com" />
+	<target host="catalog.guildeducation.com" />
+	<target host="documents.guildeducation.com" />
+		<test url="http://documents.guildeducation.com/MTP_8_Syllabus_August_2016.pdf" />
+
```
Comment (YegorIevlev) [inline src/chrome/content/rules/GuildEducation.com.xml]:
```xml
<securecookie host=".+" name=".+" />
```

File: src/chrome/content/rules/GuildEducation.com.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid Certificate:
+		pages.guildeducation.com
+-->
+<ruleset name="GuildEducation.com">
+	<target host="guildeducation.com" />
+	<target host="www.guildeducation.com" />
+	<target host="catalog.guildeducation.com" />
+	<target host="documents.guildeducation.com" />
+		<test url="http://documents.guildeducation.com/MTP_8_Syllabus_August_2016.pdf" />
```
Comment (YegorIevlev) [inline src/chrome/content/rules/GuildEducation.com.xml]: This test is not necessary.

File: src/chrome/content/rules/GuildEducation.com.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid Certificate:
+		pages.guildeducation.com
+-->
```
Comment (YegorIevlev) [inline src/chrome/content/rules/GuildEducation.com.xml]:
```xml
<!--
	Mismatched:
		- pages (CN=*.marketo.com)
-->
```

Comment (Giltyhub) [reply src/chrome/content/rules/GuildEducation.com.xml]: thanks for the review @YegorIevlev

**Pull request merged**